spec/concepts/tribunal_instance/daily_update/operation/import_spec.rb

```diff
@@ -16,9 +16,13 @@
   describe 'success' do
     before do
       create :daily_update_with_completed_units, year: '2017', month: '05', day: '17', proceeded: true
-      create :daily_update_tribunal_instance,    year: '2017', month: '05', day: '18', proceeded: false,
+      create :daily_update_tribunal_instance,
+        year: '2017',
+        month: '05',
+        day: '18',
+        proceeded: false,
         files_path: Rails.root.join('spec/fixtures/titmc/flux/2017/05/18')
-      create :daily_update_tribunal_instance,    year: '2017', month: '05', day: '19', proceeded: false
+      create :daily_update_tribunal_instance, year: '2017', month: '05', day: '19', proceeded: false
```

Comment:
Je trouve ça ignoble sans parenthèses

Reply:
en vrai il faudrait des traits et des factory plutôt que de créer des cas particulier

Reply:
Le gros avantages des cas particuliers, surtout quand tu retrouves les valeurs de `:year`, `:month` ou autre partout dans les specs qui suivent, c'est que ta pas besoin de connaitre la factory par coeur ou de faire des allers-retours entre les specs et la factory pour comprendre d'où la valeur sort.

C'est d'autant plus vrai quand les specs cassent et qu'il faut les corriger.

Sinon je parlais juste des parenthèses manquantes à l'appel de la méthode ^^
Oui je suis chiant avec ça, mais en même temps j'aime pas me priver d'éléments de syntaxe quand ils sont utiles. C'est comme les brackets, quand l'unique paramètre c'est un hash et qu'il y a pas d'ambiguïté ne pas les mettre ça ne pose pas de soucis. Quand on mélange des hash avec d'autres types d'arguments avoir les brackets autour du hash ça permet de délimiter visuellement les différents arguments.

Reply:
personnellement je n'ai aucun problème de lisibilité avec ces deux syntaxes, je pense qu'on devrait déporter le débat sur Slack

Pour les accolades : il est possible de les forcer systématiquement avec Style/BracesAroundHashParameters
Pour les parenthèses : il est possible de les forcer systématiquement pour `create :factory` et tout le reste avec `IgnoreMacros: false` dans Style/MethodCallWithArgsParentheses. Mais... ça veut dire mettre aussi des parenthèses à : `describe, context, it, include, it_behaves_like, factory, tous les mots clefs active record & Rails etc...` euh non...

Reply:
> Mais... ça veut dire mettre aussi des parenthèses à : describe, context, it, include, it_behaves_like, factory, tous les mots clefs active record & Rails etc... euh non...

C'est faux. Confère le commit 2229fe (c'est un dev que j'ai fait sur ta PR Rubocop) sur le repo admin_apientreprise.